consensus/service/src/config/mod.rs

```diff
@@ -101,6 +101,10 @@ pub struct Config {
     /// Allow extreme (>= 1MOB, <= 0.000_000_01 MOB).
     #[structopt(long)]
     pub allow_any_fee: bool,
+
+    /// The configured block version
+    #[structopt(long, env = "MC_BLOCK_VERSION", default_value = "1")]
+    pub block_version: u32,
```

Comment:
What happens if you change this to be of type `BlockVersion`?

Reply:
Will try